### ISSUE TITLE
<issue>There is no option to select the Speed ​​model after deploying in Vercel like morphic.sh.B

### ISSUE THREAD

**Issue by stephonye** [miurla/morphic#276]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I tried to set OPENAI_API_MODEL = gpt-4o-mini,gpt-4o in the Environment Variables of the Vervel project, but from OpenAI's API Usege, I saw that it is still billed using GPT-4 Trubo, and there is no model selection like morphic.sh. Do I need to set it somewhere?

### Expected Behavior

The API calls GPT-4o/GPT-4o-mini, and i can select Speed ​​or other models before searching.

### Steps To Reproduce

Reproducible at any time

### Environment

```markdown
- OS:macOS Sonoma 14.5
- Browser:Safari 17.5 and Chrome 126.0.6478.183
```

**Comment by miurla** [comment]
> I tried to set OPENAI_API_MODEL = gpt-4o-mini,gpt-4o in the Environment Variables of the Vervel project

You cannot set multiple models in the environment variables.

> I saw that it is still billed using GPT-4 Turbo

In the latest code, `GPT-4 Turbo` will not be used unless it is set in the environment variables. Please debug the value of `openaiApiModel` locally.

https://github.com/miurla/morphic/blob/main/lib/utils/index.ts#L19

> there is no model selection like morphic.sh

It has not been implemented yet: https://github.com/miurla/morphic/issues/259

**Comment by somehow22** [comment]
> > I tried to set OPENAI_API_MODEL = gpt-4o-mini,gpt-4o in the Environment Variables of the Vervel project
> 
> You cannot set multiple models in the environment variables.
> 
> > I saw that it is still billed using GPT-4 Turbo
> 
> In the latest code, `GPT-4 Turbo` will not be used unless it is set in the environment variables. Please debug the value of `openaiApiModel` locally.
> 
> https://github.com/miurla/morphic/blob/main/lib/utils/index.ts#L19
> 
> > there is no model selection like morphic.sh
> 
> It has not been implemented yet: #259

Hi, I just deployed morphic with searxng, and it works! though I am looking for an option to select model like morphic.sh, but nowhere to find in settings or env. But I noticed you marked this as `completed` months ago, any documentation illustrating this feature?

**Comment by miurla** [comment]
@somehow22 I implemented this in #371.
